api/hasura/actions/_handlers/allocationCsv.ts

```diff
@@ -4,7 +4,8 @@ import type { VercelRequest, VercelResponse } from '@vercel/node';
 import { v4 as uuidv4 } from 'uuid';
```

Comment:
@zashton can you please write a jest test for this, to verify that when you add certain inputs, they show up in the CSV? you can see an example of how to mock the request & response objects as well as the hasura query [here](https://github.com/coordinape/coordinape/blob/main/api/hasura/cron/_recoverVaults.test.ts) if needed. thanks!

Reply:
it may make sense to have a few different tests to cover the different cases (gift, fixed, combined)